docs/How-To/cli/README.md

```diff
@@ -11,35 +11,54 @@ This document provides a high-level overview of how the command line (CLI) inter
+    - [Vestings](#vestings)
 
 ## Transaction
 
 +++ https://github.com/tendermint/fundraising/blob/main/proto/fundraising/tx.proto#L14-L29
 
 ### CreateFixedPriceAuction
 
-This command provides an opportunity for an auctioneer to create an auction to raise funds for their project. It is the most basic and simple type of an auction that has first come first served basis. When an auctioneer creates a fixed price auction, they must determine the fixed starting price. It is proportional to the paying coin denom. To give you an example, an auctioneer sells X coin and plans to receive Y coin for the auction. The price of X coin is determined by the proportion of Y coin. Let's assume that the price of Y coin is currently $30 and the auctioneer wants to sell their X coin for $15, then they must set 0.5 as the fixed starting price. Once the auction is successfully created, bidders can now start to bid. The bidders must provide the same start price when they bid. See the [spec](https://github.com/tendermint/fundraising/blob/main/x/fundraising/spec/01_concepts.md#auction-types) for a detailed and technical information about the fixed price auction type.
+An auctioneer can create a fixed price auction by setting the following parameters. In a fixed price auction, `start_price` is the matched price and bidders can buy the selling coins on a first-come, first-served basis. See the [spec](https://github.com/tendermint/fundraising/blob/main/x/fundraising/spec/01_concepts.md#auction-types) for a detailed and technical information about a fixed priced auction type.
 
-JSON example:
+Usage
+```bash
+create-fixed-price-auction [file]
+```
+
+
+Field description of the input file
+
+| **Field**         |  **Description**                                                                    |
+| :---------------- | :---------------------------------------------------------------------------------- |
+| allowed_bidders | The list of allowed bidders that can participate in the auction, with a maximum possible bid amount for each bidder. It is empty when an auction is created. The module is designed to delegate permission to an external module to add its allowed bidders to the auction. |
+| start_price       | The starting price of the selling coin; it is proportional to the paying coin denom. This is the matched price. | 
+| selling_coin      | The selling amount of coin for the auction                                      | 
+| paying_coin_denom | The paying coin denom that bidders use to bid with                                  | 
+| vesting_schedules | The vesting schedules that release the paying coins to the autioneer                | 
+| start_time        | The start time of the auction                                                       | 
+| end_time          | The end time of the auction                                                         | 
+|                   |                                                                                     |
```

Comment:
I thought this line was needed to close the table, but it looks like it is adding another empty row. Can you delete this? https://github.com/jeonghoyeo7/fundraising/tree/128-fix-cli-placeBid/docs/How-To/cli#createfixedpriceauction
```suggestion
```

Reply:
Removed.